### PR TITLE
Hygiene: Always use https for wikimedia/wmflabs URLs in example configs

### PR DESCRIPTION
**Pull request by mdholloway**

### CODE REVIEW

File: config.example.wikimedia.yaml [hunk 1]
```diff
@@ -17,27 +17,27 @@ default_project: &default_project
...
         parsoid:
-          host: http://parsoid-beta.wmflabs.org
+          host: https://parsoid-beta.wmflabs.org
           grace_ttl: 1
         action:
           apiUriTemplate: "{{'https://{domain}/w/api.php'}}"
           baseUriTemplate: "{{'https://{domain}/api/rest_v1'}}"
         graphoid:
-          host: http://graphoid.wikimedia.org
+          host: https://graphoid.wikimedia.org
```
Comment (jdforrester) [inline config.example.wikimedia.yaml]: Not routable?

Comment (d00rman) [reply config.example.wikimedia.yaml]: Indeed, this doesn't exist any more. Let's change this to `graphoid-beta.wmflabs.org`

**Comment by Pchelolo** [comment]
If we're touching this, we can also explicitly prohibit http in x-request-filters around here https://github.com/wikimedia/restbase/pull/1018/files#diff-541b6e195e9da580fc0bd42761ee761fR88

(remove `?` from the `https?` regex)

**Comment by Pchelolo** [comment]
Let's see what travis says, but I don't anticipate and tests to fail since they use test config. Did you try manually using the example config and making some random requests to RB? Does it still work?

**Comment by mdholloway** [comment]
Yes, I tried out several requests to various endpoints and all seems fine.

**Comment by Pchelolo** [comment]
Cool. Thank you @mdholloway

**Pull request merged**